src/components/gearSelection.js

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+import Mask from '../icons/gear_types/mask.png'
+import Backpack from '../icons/gear_types/backpack.png'
+import Vest from '../icons/gear_types/vest.png'
+import Glove from '../icons/gear_types/gloves.png'
+import Holster from '../icons/gear_types/holster.png'
+import Kneepads from '../icons/gear_types/kneepads.png'
+import Rifle from '../icons/weapons/Rifle.png'
+import './builder.css'
+
+function GearSelection() {
+    return (
+        <div className="createBuildGrid">
+            <div className="card signature">
+                Placeholder Text
+            </div>
+            <div className="card gernade">
+                
+                Placeholder Text
+            </div>
+            <div className="card weapon">
+                <img className="icon" src={Rifle} alt="primary"/>
+                <span className="name named">Baker's Dozen</span>
```

Comment:
I assume these will be selects populated from the data set?

Reply:
Yeah this is just a placeholder for me to play with the styling.